main.py

```diff
@@ -71,19 +64,30 @@ def showMap(city):
 
 @app.route('/', methods=['GET', 'POST'])
 def index():
-    cityName = getRandomCityName(POPULATION_THRESHHOLD)
-    cityCoords = getCityCoordinatesByName(cityName)
-    city = City(cityName, cityCoords)
+    city = getRandomCity()
     mapOfCity = showMap(city)
+    correctAnswer = city.cityId
+
+    return render_template('index.html', mapOfCity=mapOfCity, correctAnswer=correctAnswer)
+
+@app.route('/guess', methods=['POST'])
+def checkAnswer():
+    req = request.get_json()
     
-    return render_template('index.html', mapOfCity=mapOfCity)
-    
+    if request.method == 'POST':
+        if str(req['attempt']).title() != connectDb.checkAnswer(req['cityId']):
+            isCorrect = req['isCorrect']['no']
+            res = make_response(jsonify(isCorrect), 200)
+            return res
+
+                
+        isCorrect = req['isCorrect']['yes']
+        res = make_response(jsonify(isCorrect), 200)
+        return res
+
+    return res
```

Comment:
You duplicate so much code here. Really strive to not do that. Could be rewritten as follows:

```python
  isCorrect = str(req['attempt']).lower() != connectDb.checkAnswer(req['cityId']).lower()
  return make_response(jsonify({ "correct": isCorrect )), 200)
```

As for `title` vs `lower`. What if the country is `United States of America` and the user put `united states of America`. 
Then you'd be comparing `United states of America` (no capital S) to `Unites States Of America` which evaluates to false. Always put the strings on an even playing field and go lower case.